base_data_import/__init__.py

```diff
@@ -1,2 +1,4 @@
+# -*- coding: utf-8 -*-
+
```

Comment:
No need to revive these lines.

Reply:
Yes. I already created like this 4 PR. For the remaining others, I won't revive those lines again.